docs/userguide/checkpoints.rst

```diff
@@ -18,46 +18,92 @@ will checkpoint all results that have updated since the last checkpoint. When lo
 checkpoints, if entries with results from multiple functions (with identical hashes)
 are encountered, only the last entry read will be considered.
 
-Checkpointing comes with the same caveats as AppCaching but with one key
-difference: each checkpointing event is manually triggered by the user.
-Checkpoints are loaded from checkpoint files at the start of the
-DataFlowKernel and written out to checkpoint files only when the
-``DataFlowKernel.checkpoint()`` function is invoked. The checkpoint is written
-out to the ``runinfo/RUN_ID/checkpoint/`` directory.
+Checkpointing works in the following modes :
 
-Example code follows:
+1. ``task_exit`` : In this mode, each app is checkpointed when it has completed or
+   failed (after retries if enabled). This mode reduces the risk of losing information
+   from completed tasks to a minimum.
+
+   >>> config["globals"]["checkpoint"] = 'task_exit'
```

Comment:
Let's consider reserving the use of `>>>`. It is the Python interpreter prompt and the standard way of running [doc tests](https://docs.python.org/2/library/doctest.html) so people may think they should be able to cut and paste directly into a prompt to test something. By not using this prompt here we avoid possible confusion and could eventually support doc tests.

Reply:
This is paste'able into the interpreter and is valid code. I don't understand the problem.
What I want to do is hilight code snippets assuming that users will copy paste it.